cc3d/twedit5/Plugins/CC3DProject/unzipper.py

```diff
@@ -0,0 +1,91 @@
+import os.path
```

Comment:
I know this file is duplicated and that's ok for now but I also wonder if perhaps this means that we need to created a fourth package (core_utils or something like that) that would be shared between core cc3d code, player, twedit++. core_utils would include functions like unzipper, configuration, and anything that is trully shared and does not depend on PyQt. This is just a thought

Reply:
I'd had this same thought. Maybe the core package really should be just the computing, and a utilities package should contain all auxiliary functionality like configurations, zipping, etc.